packages/jaeger-ui/src/components/Monitor/ServicesView/index.tsx

```diff
@@ -210,6 +225,7 @@ export class MonitorATMServicesViewImpl extends React.PureComponent<TPropsWithIn
 
     if (currentService) {
       this.endTime = Date.now();
+      store.set('lastAtmSearchSpanKind', selectedSpanKind);
```

Comment:
When we change this to SPM, a couple more improvements:
- use constants instead of raw strings 
- keep abbreviations in uppercase, SPM not Spm (but constants should probably all be named in uppercase)